cmd/swagger/swagger.go

```diff
@@ -33,6 +33,15 @@ var opts struct {
 }
 
 func main() {
+	// Recovering from internal panics
+	defer func() {
```

Comment:
this changes makes it so there is no more stacktrace, i don't think that is what we want

Reply:
The point is to catch errs from methods which do not return error.
My concern was essentially the key "baseImport()" method.
I can let the panic go if you prefer. I suggest to leave it as a comment: best to catch the stack hen the product is more stable.